shared/build.gradle.kts

```diff
@@ -37,27 +39,56 @@ kotlin {
                 implementation(compose.material)
                 @OptIn(org.jetbrains.compose.ExperimentalComposeLibrary::class)
                 implementation(compose.components.resources)
+                api("dev.icerock.moko:mvvm-core:0.15.0")
+                api("io.ktor:ktor-client-core:2.2.4")
+                api("dev.icerock.moko:network:0.20.1")
+                api("dev.icerock.moko:resources:0.20.1")
+//                api("dev.icerock.moko:resources-compose:0.20.1")
+
+                //Could not resolve all dependencies for configuration ':shared:iosSimulatorArm64CompileKlibraries'.
```

Comment:
library not support iosSimulator? can you check it in repo? and add issue link here